### ISSUE TITLE
Documentation for FindTheGoal

### ISSUE THREAD

**Issue by rocanaan** [crowdAI/marLo#55]
Hello,

I was looking at the documentation for the environments and realized the [FindTheGoal environment](https://github.com/crowdAI/marLo/blob/master/marlo/envs/FindTheGoal/main.py) is one of the few environments with no documentation.

Since this is the environment that will be used for the first phase, I would like to know if there is an ETA for the documentation on this environment to be made available.

If I may make some suggestions, I would like to know:
- If the room is always flat or if the goal may be occluded by walls (making it somewhat like a maze)
- What materials are possible for the goal block and for the rest of the room
- If there might be obstacles such as lava, holes, steps, etc.

Thank you very much!

**Comment by rocanaan** [comment]
#40

**Comment by spMohanty** [comment]
@rocanaan : You are right, there was a small issue. While I fixed it, I realised the docstirngs for FindTheGoal are not written. @idvhfd confirmed that he will be adding them soon.

And wrt your questions,

* the room will always be flat, and goal will not be occluded by walls
* the materials for goal block could be : `gold_ore diamond_ore redstone_ore`
* there will be no obstacles.

**Comment by spMohanty** [comment]
@justinglibert : Its defined here. I might be misinterpreting it. 
@rdgain : Can you comment on this ?

**Comment by justinglibert** [comment]
@spMohanty what I mean is that when I run the environment I never see any block other than the gold one. I looked at the mission file and didn't find a parameter to set that up. If during evaluation our agent can encounter non gold block, how are we supposed to set our local environment to reproduce this diversity?

**Comment by idvhfd** [comment]
Apologies for the delay, I was unable to attend this earlier. I added a brief description in the same style as the previous ones (awaiting confirm and pull). Thanks for pointing this out and please do accept my apologies for creating this situation. Let me know if anything else like this is needed.

@justinglibert the seed is being passed as a parameter, that's where the block comes from. Waiting on @spMohanty for exact path to that bit as I can't look for it right now.
